Patches/Revia Race/ThingDefs_Misc/Weapons_Guns.xml

```diff
@@ -0,0 +1,634 @@
+              <value>
+                <tools>
+                  <li Class="CombatExtended.ToolCE">
+                    <label>handle</label>
+                    <capacities>
+                      <li>Blunt</li>
+                    </capacities>
+                    <power>8</power>
+                    <cooldownTime>1.47</cooldownTime>
+                    <armorPenetrationBlunt>0.19</armorPenetrationBlunt>
+                    <linkedBodyPartsGroup>Handle</linkedBodyPartsGroup>
+                  </li>
+                  <li Class="CombatExtended.ToolCE">
+                    <label>point</label>
+                    <capacities>
+                      <li>Stab</li>
+                    </capacities>
+                    <power>13</power>
+                    <cooldownTime>1.32</cooldownTime>
+                    <chanceFactor>1.165</chanceFactor>
+                    <armorPenetrationBlunt>0.31</armorPenetrationBlunt>
+                    <armorPenetrationSharp>0.62</armorPenetrationSharp>
+                    <linkedBodyPartsGroup>Point</linkedBodyPartsGroup>
+                  </li>
+                  <li Class="CombatExtended.ToolCE">
+                    <label>edge</label>
+                    <capacities>
+                      <li>Gash</li>
+                    </capacities>
+                    <power>11</power>
+                    <cooldownTime>1.3</cooldownTime>
+                    <chanceFactor>1.243</chanceFactor>
+                    <armorPenetrationBlunt>0.21</armorPenetrationBlunt>
+                    <armorPenetrationSharp>0.35</armorPenetrationSharp>
+                    <linkedBodyPartsGroup>Edge</linkedBodyPartsGroup>
+                  </li>
+                </tools>
+              </value>
+            </li>
+            <li Class="PatchOperationAdd">
+              <xpath>Defs/ThingDef[defName="ReviaBlessedSerratedDagger"]/statBases</xpath>
+              <value>
+                <Bulk>1.5</Bulk>
+                <MeleeCounterParryBonus>0.33</MeleeCounterParryBonus>
+              </value>
+            </li>
+            <li Class="PatchOperationAdd">
+              <xpath>Defs/ThingDef[defName="ReviaBlessedSerratedDagger"]</xpath>
+              <value>
+                <equippedStatOffsets>
+                  <MeleeCritChance>1</MeleeCritChance>
+                  <MeleeParryChance>0.6</MeleeParryChance>
+                  <MeleeDodgeChance>0.1</MeleeDodgeChance>
+                </equippedStatOffsets>
+              </value>
+            </li>
+            <!-- ==========  Scourge  =========== -->
+            <li Class="PatchOperationReplace">
+              <xpath>Defs/ThingDef[defName="ReviaSerratedSword"]/tools</xpath>
+              <value>
+                <tools>
+                  <li Class="CombatExtended.ToolCE">
+                    <label>handle</label>
+                    <capacities>
+                      <li>Blunt</li>
+                    </capacities>
+                    <power>9</power>
```

Comment:
This is, and a lot of the other blunt attacks with the melee weapons, are pretty high.